extensions/kube-object-event-status/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kube-object-event-status",
-  "version": "0.1.0",
+  "version": "0.0.1",
```

Comment:
Why do you need this? Something still fishy with the rebase?

Reply:
yes fishy indeed.